Content.Shared/_DV/Projectiles/SimpleForeignBodyContainerComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._DV.Projectiles;
+
+/// <summary>
+/// When a projectile attempts to embed into this entity, store it on this entity itself
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class SimpleForeignBodyContainerComponent : Component
+{
+    [DataField]
+    public string ContainerName = "internally_embedded_projectiles";
+}
```

Comment:
remove, have ForeignBodyContainerComponent handle them directly and have the bodycomponent event handler just relay it to the targeted or a random part

Reply:
this is for non-BodyComponent mobs and honestly could probably be removed